### PR TITLE
fcntl F_DUPFD_CLOEXEC has different values on Solaris and Illumos

### PR DESCRIPTION
**Pull request by gco**

### CODE REVIEW

File: src/unix/solarish/illumos.rs [hunk 1]
```diff
@@ -42,6 +42,9 @@ pub const F_OFD_SETLKW: ::c_int = 49;
 pub const F_FLOCK: ::c_int = 53;
 pub const F_FLOCKW: ::c_int = 54;
 
+// fcntl F_DUPFD_CLOEXEC has different values on Solaris and Illumos
```
Comment (jclulow) [inline src/unix/solarish/illumos.rs:45]:
```suggestion
```

I would remove the comments.  The separate file structure already denotes different constants for illumos and Solaris, and the file would be full of comments like this if we noted every difference.

File: src/unix/solarish/solaris.rs [hunk 2]
```diff
@@ -36,6 +36,9 @@ pub const TCP_KEEPIDLE: ::c_int = 0x1d;
 pub const TCP_KEEPCNT: ::c_int = 0x1e;
 pub const TCP_KEEPINTVL: ::c_int = 0x1f;
 
+// fcntl F_DUPFD_CLOEXEC has different values on Solaris and Illumos
```
Comment (jclulow) [inline src/unix/solarish/solaris.rs:39]:
```suggestion
```

**Review by jclulow — CHANGES_REQUESTED** [review]

**Review by jclulow — APPROVED** [review]
Looks good to me -- thanks!

**Review by JohnTitor — APPROVED** [review]

**Pull request merged**